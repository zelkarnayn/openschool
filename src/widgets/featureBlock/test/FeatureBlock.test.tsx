import {cleanup, render, screen} from "@testing-library/react";
import BannerFeature from "../ui";
import {it} from "vitest";

afterEach(cleanup)

test("Render feature banner", () => {
    render(<BannerFeature/>)
    expect(true).toBeTruthy()
})

it('test', () => {
    render(<BannerFeature/>)
    const we = screen.queryByText(/we/i)
    expect(we).toMatchSnapshot()
})
