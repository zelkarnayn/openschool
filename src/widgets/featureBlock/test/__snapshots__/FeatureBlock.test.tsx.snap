// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot Feature Banner 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_feature__wrapper_347dc9"
      >
        <div
          class="_feature__main_347dc9"
        >
          <div
            class="_feature__block_347dc9"
          >
            <h1>
              10K+
            </h1>
            <p>
              Satisfied Costumers
              <br />
              All Great Over The World
            </p>
          </div>
        </div>
        <div
          class="_feature__main_347dc9"
        >
          <div
            class="_feature__block_347dc9"
          >
            <h1>
              4M
            </h1>
            <p>
              Healthy Dishes Sold 
              <br />
               Including Milk Shakes Smooth
            </p>
          </div>
        </div>
        <div
          class="_feature__main_347dc9"
        >
          <div
            class="_feature__block_347dc9"
          >
            <h1>
              99.99%
            </h1>
            <p>
              Reliable Customer Support 
              <br />
              We Provide Great Experiences
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_feature__wrapper_347dc9"
    >
      <div
        class="_feature__main_347dc9"
      >
        <div
          class="_feature__block_347dc9"
        >
          <h1>
            10K+
          </h1>
          <p>
            Satisfied Costumers
            <br />
            All Great Over The World
          </p>
        </div>
      </div>
      <div
        class="_feature__main_347dc9"
      >
        <div
          class="_feature__block_347dc9"
        >
          <h1>
            4M
          </h1>
          <p>
            Healthy Dishes Sold 
            <br />
             Including Milk Shakes Smooth
          </p>
        </div>
      </div>
      <div
        class="_feature__main_347dc9"
      >
        <div
          class="_feature__block_347dc9"
        >
          <h1>
            99.99%
          </h1>
          <p>
            Reliable Customer Support 
            <br />
            We Provide Great Experiences
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
